src/curated-corpus/components/ProspectListCard/ProspectListCard.tsx

```diff
@@ -69,8 +69,13 @@ export const ProspectListCard: React.FC<ProspectListCardProps> = (
     onReject,
   } = props;
 
-  const timeToRead = parserItem.timeToRead ? (
-    `${parserItem.timeToRead} min(s)`
+  let _parserItem = parserItem;
+  if (!parserItem) {
+    _parserItem = {};
+  }
+
+  const timeToRead = _parserItem.timeToRead ? (
+    `${_parserItem.timeToRead} min(s)`
```

Comment:
Can you please explain what lines 72-75 are intended for? 

I think lines 77-81 here are essentially doing the same thing as what we had before 🤔

Reply:
if  `_parserItem`  - this causes the UI to crash on line 77.  Oh we can add a ? there

Reply:
Was it causing issues how we had it before? 

If `parserItem` object is `null/undefined` and/or `timeToRead` property on that object is `null/undefined/falsey` (it will be if the `parserItem` itself is falsey), then the ternary expression would default to the clause after the ` : `. 

So line `74` doesn't really do anything here 🤔 unless I'm missing something

Reply:
@Herraj @jsrawan-mobo is this for when running the tool locally (along with prospect-api + admin-api locally) & the parserItem is null?

Reply:
Yes this happens locally all the time.  It could happen in production also.

if parserItem is undefined,  line 77 throws because you can't dereference.  I added a test to illustrace.  Lines 137 later on had the right check.

Reply:
> So line `74` doesn't really do anything here 🤔 unless I'm missing something

you can't  do `undefined.attr` or `null.attr` you can do `false.attr` 
I think your meant chaining

`undefined' && undefined.attr1 && undefined.attr1.attr2`

is same as  `undefined?.attr1?.attr2`